yamls/extension/test-extension-deployment.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: runtime-extension-test
+  name: test-extension-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: runtime-extension-test
+  name: test-extension-manager
+  namespace: test-extension-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: runtime-extension-test
+  name: test-extension-manager-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
+  - update
+  - create
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: runtime-extension-test
+  name: test-extension-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: test-extension-manager-role
+subjects:
+- kind: ServiceAccount
+  name: test-extension-manager
+  namespace: test-extension-system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: runtime-extension-test
+  name: test-extension-webhook-service
+  namespace: test-extension-system
+spec:
+  ports:
+  - port: 443
+    targetPort: webhook-server
+  selector:
+    app: test-extension-manager
+    cluster.x-k8s.io/provider: runtime-extension-test
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: runtime-extension-test
+  name: test-extension-manager
+  namespace: test-extension-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test-extension-manager
+      cluster.x-k8s.io/provider: runtime-extension-test
+  template:
+    metadata:
+      labels:
+        app: test-extension-manager
+        cluster.x-k8s.io/provider: runtime-extension-test
+    spec:
+      containers:
+      - command:
+        - /manager
+        image: gcr.io/kakaraparthy-devel/test-extension:v1.0.0
```

Comment:
@ykakarap is this image multi-arch?

Reply:
Probably we don't need it multi-arch

Reply:
Container is working on Mac ARM (not sure if it's multi-arch or not now), but the flow is working fine as is.

Reply:
Shouldn't be multi-arch

Reply:
Interestingly - the image is AMD64, the docker VM under docker desktop is ARM and the Mac is ARM, but the image works :slightly_smiling_face: Docker run initially complains that the arch doesn't match, but it then it runs the image. Rosetta is pretty magic.